### PR TITLE
Ofl license without rfn

### PR DESCRIPTION
**Pull request by andreaslarsen**
Removed the reserved font name as discussed in #2184

### CODE REVIEW

File: fontforge/ofl.c [hunk 1]
```diff
@@ -6,8 +6,7 @@
 #include "ofl.h"
 
 static char *ofl_eng[] = {
-    "Copyright (c) %1$d, %2$s (<URL|email>),",
-    "with Reserved Font Name %s.",
```
Comment (adrientetar) [inline fontforge/ofl.c]: You need to remove the corresponding printing code as well.

**Comment by adrientetar** [comment]
Please drop the merge commits as well.

**Comment by andreaslarsen** [comment]
I can fix the merge commits but wouldn't know how to remove the corresponding printing code - sounds like something you can do in a split second so I'm just closing this ;)

**Comment by adrientetar** [comment]
It needs to be refactored to follow [this](https://github.com/adobe-fonts/source-serif-pro/issues/3#issuecomment-82096174) anyway. I'll have a look when I have time.

**Pull request closed without merging**